docs/source/data-publishing/ogcapi-edr.rst

```diff
@@ -58,12 +58,29 @@ The `xarray-edr`_ provider plugin reads and extracts `NetCDF`_ and `Zarr`_ data
          format:
             name: zarr
             mimetype: application/zip
+    
+    providers:
+       - type: edr
+         name: xarray-edr
+         data: s3://power-analysis-ready-datastore/power_901_annual_meteorology_utc.zarr
+         format:
+            name: zarr
+            mimetype: application/zip
+         storage_options:
```

Comment:
Can we rename to `options` for some consistency with other providers?  And then namespace with `s3`?  So:

```
options:
  s3:
    anon: true
    requestor_pays: false
```